docs/trex_contributing.md

```diff
@@ -16,15 +16,13 @@ Submitting to the Community Portal {#portal}
 
 1. Create a fork of the `extensions-api` repository.  
 
-2. In your fork of the repository, checkout the `submissions` branch.
+1. In your fork of the repository, find the community folder at the root of the repository. 
 
-3. In your fork of the repository, find the community folder at the root of the repository. 
+1. In the `community_extensions.json` file, add an entry for your extension.  Name, author, and description are required. All other fields are optional.  While we recommend both providing the source code and a hosted version of your extension, you may choose one or the other.  If you choose to provide a hosted version of your extension, see the next step (skip otherwise).
 
-4. In the `community_extensions.json` file, add an entry for your extension.  Name, author, and description are required. All other fields are optional.  While we recommend both providing the source code and a hosted version of your extension, you may choose one or the other.  If you choose to provide a hosted version of your extension, see step 5 (skip otherwise).
+1. First, host your extension on a web server (see below for suggestions). Second, create a manifest file (`.trex`) where the URL points to the hosted location of your extension.  Finally, in your pull request, place the manifest file in the `community/CommunityManifests` folder.  Also place the file name into your entry in community_extensions.json.  Ensure the filename matches your manifest file and it is unique from others in the folder.
 
-5. First, host your extension on a web server (see below for suggestions). Second, create a manifest file (`.trex`) where the URL points to the hosted location of your extension.  Finally, in your pull request, place the manifest file in the `community/CommunityManifests` folder.  Also place the file name into your entry in community_extensions.json.  Ensure the filename matches your manifest file and it is unique from others in the folder.
-
-6. Submit a pull request from the submissions branch of your fork to the submissions branch in the official repository.
+1. Submit a pull request from your fork to the master branch in the official repository.
```

Comment:
Do we still want them to open a PR to master?
Answer: Yes.

Reply:
Yes. it's simpler for them I believe